cfme/provisioning.py

```diff
@@ -35,6 +40,136 @@ def select_security_group(sg):
     sel.sleep(1)
 
 
+class ProvisioningForm(BaseLoggedInPage):
```

Comment:
@izapolsk I'm thinking I should maybe pull this into a separate PR that we can review much faster that all the instance stuff. thoughts?